lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -85,10 +85,10 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     %endif
 
                     %if linked_in_url:
-                    <a class="action action-linkedin-profile btn btn-overlay btn-small icon-only" id="action-share-linkedin" target="_blank" href="${linked_in_url}" title="${_('Add to LinkedIn Profile')}" data-course-id="${course_id}" data-certificate-mode="${course_mode}">
+                    <button class="action action-linkedin-profile btn btn-overlay btn-small icon-only" id="action-share-linkedin" onclick="window.open('${linked_in_url}');" title="${_('Add to LinkedIn Profile')}" data-course-id="${course_id}" data-certificate-mode="${course_mode}">
```

Comment:
Hmm, it'd be great if we didn't attach the onclick handler as a DOM attribute here. I think our standards are to move that logic into the JS methods/templates layer and target a specific element class or attribute value there. This not only separates out behavioral logic into one place but it also lets us handle other events rather than just click. @andy-armstrong might have more thoughts on this.